universal-login-example/src/components/Backup.js

```diff
@@ -1,14 +1,74 @@
 import React, { Component } from 'react';
 import BackupView from '../views/BackupView';
 import PropTypes from 'prop-types';
+import { toWords } from '../Daefen';
+import ethers from 'ethers';
+import DEFAULT_PAYMENT_OPTIONS from '../../config/defaultPaymentOptions';
 
 class Backup extends Component {
+  constructor(props) {
+    super(props);
+    this.identityService = this.props.services.identityService;
+    this.sdk = this.props.services.sdk;
+    this.state = {
+      backupCodes: ['loading...', 'loading...', 'loading...'],
```

Comment:
I never see those. It feels "too fast" for me. Is it just a question of my machine?

Reply:
Oh I see what is happening! More below